tripper/literal.py

```diff
@@ -57,7 +64,8 @@ def __new__(cls, value, lang=None, datatype=None):
             elif isinstance(value, float):
                 string.datatype = XSD.double
             elif isinstance(value, (bytes, bytearray)):
-                string = value.hex()
+                string = super().__new__(cls, value.hex())
```

Comment:
Can you add a one line comment in the code why this is necessary?

Reply:
I think I understand and agree, but I think this is a case that a very brief comment might help future software-spelunkers grok the logic

Reply:
Done